EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvPredicateImpl.swift

```diff
@@ -21,9 +21,54 @@ import Foundation
 
 struct EvolvPredicateImpl {
     
-    public func evaluate(context: Codable, predicate: Codable) {
+    
+    
+    struct FilterOperations {
+        static func contains (a: String, b: String) -> Bool {
+            return a.contains(b)
+        }
+        
+        static func defined (a: String?) -> Bool {
+            return a != nil
+        }
+        
+        static func equal<T: Equatable> (a: T, b: T) -> Bool {
+            return a == b
+        }
+        
+        static func exists (a: String) -> Bool {
```

Comment:
This should be verifying the the key exists in the context - returning true feels incorrect here

Reply:
Exactly. Forgot to change the implementation